spec/views/documents/history.html.erb_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe "documents/history.html.erb" do
+  describe "document history" do
+    let(:edition) { build(:edition) }
+
+    it "shows the timeline entry" do
+      create(:timeline_entry, edition: edition)
+      assign(:edition, edition)
+      assign(:timeline_entries, edition.timeline_entries.page(1).per(50))
+      expect(render)
+        .to include(I18n.t("documents.history.entry_types.created"))
+    end
+
+    it "shows a Whitehall timeline entry" do
+      whitehall_migration_timeline_entry = create(:whitehall_migration_timeline_entry)
+      create(:timeline_entry,
+             edition: edition,
+             entry_type: "whitehall_migration",
+             details_type: "WhitehallMigration::TimelineEntry",
+             details_id: whitehall_migration_timeline_entry.id)
+      assign(:edition, edition)
+      assign(:timeline_entries, edition.timeline_entries.page(1).per(50))
+      expect(render)
+        .to include(I18n.t("documents.history.entry_types.whitehall_migration.new_edition"))
+        .and include(whitehall_migration_timeline_entry.body)
+    end
+  end
+end
```

Comment:
I think if you had a separate view this might be easier to test as you'd have to do less DB writes to set-up creation and could focus on an individual entry.